### PR TITLE
add section to zar README on multi-key search

### PR DESCRIPTION
**Pull request by mccanne**

### CODE REVIEW

File: cmd/zar/README.md [hunk 1]
```diff
@@ -325,6 +325,66 @@ zq "id.orig_h=10.164.94.120" zng/*.gz | zq -f table "count() by _path" -
...
+```
+zar find -z -x custom2 216.58.193.206 | zq -t -
+```
+and of course you can sum up all the response bytes to get a table and output
+it as text...
+```
+zar find -z -x custom2 216.58.193.206 | zq -f text "sum(resp_bytes)" -
+```
+While you can't "wild card" the primary key when doing a search, remember
+everything is a zng file, so you an do a brute-force search on the indexes,
```
Comment (philrz) [inline cmd/zar/README.md:369]:
```suggestion
everything is a zng file, so you can do a brute-force search on the indexes,
```

File: cmd/zar/README.md [hunk 2]
```diff
@@ -325,6 +325,66 @@ zq "id.orig_h=10.164.94.120" zng/*.gz | zq -f table "count() by _path" -
...
+any value for the secondary key, e.g.,
+```
+zar find -z -x custom2 216.58.193.206 | zq -t -
+```
+and of course you can sum up all the response bytes to get a table and output
+it as text...
+```
+zar find -z -x custom2 216.58.193.206 | zq -f text "sum(resp_bytes)" -
+```
+While you can't "wild card" the primary key when doing a search, remember
```
Comment (philrz) [inline cmd/zar/README.md:368]: Is this "While you can't..." just a statement of us not having implemented something yet? This theme of "everything is just a ZNG file" runs throughout this doc, and I can do searches with wild cards on other fields in ZNG files, so as a reader it's not clear to me why this limitation exists.

File: cmd/zar/README.md [hunk 3]
```diff
@@ -325,6 +325,66 @@ zq "id.orig_h=10.164.94.120" zng/*.gz | zq -f table "count() by _path" -
...
+everything is a zng file, so you an do a brute-force search on the indexes,
+e.g., to look aggregate across the secondary keys.  Let's say we wanted
```
Comment (philrz) [inline cmd/zar/README.md:370]: It seems there's one or more words missing here. Did you mean to say "to compute an aggregation across the secondary keys"?

File: cmd/zar/README.md (hunk 2 above)
Comment (mccanne) [reply cmd/zar/README.md:368]: I'll clarify, thanks.  The search index is sorted primary-key first etc, so you can't wildcard the primary key in "zar find" and locate all the secondary keys with an efficient b-tree lookup. but you can brute-force search the base-layer zng-file index with a match on the secondary key.

**Pull request merged**